apcd-cms/src/apps/admin_extension/static/admin_extension/css/modal.css

```diff
@@ -1,11 +1,39 @@
 dt {
```

Comment:
Can you leverage the common modal styles in the taccsite_custom/apcd_cms directory? Similar to how you did for your WP-151 PR?

Reply:
Yes definitely!